contracts/contracts/ChatOracle.sol

```diff
@@ -245,6 +247,11 @@ contract ChatOracle {
         latestAttestationOwner = msg.sender;
     }
 
+    function addPcr0Hash(string memory pcr0Hash) public onlyOwner {
```

Comment:
onlyOwner is correct here yep?

Reply:
yes!

how can we read the pcr0Hashes on the browser/client side?

Reply:
the mapping and "latestPcr0HashOwner" are public, so we can query latestPcr0HashOwner and then pcr0Hashes[latestPcr0HashOwner]